src/style.css

```diff
@@ -0,0 +1,113 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  height: 300px;
+  min-width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+}
+
+.box-1 {
+  background-color: rgb(100%, 0%, 0%);
+  order: 1;
+}
+
+.box-2 {
+  background-color: rgb(80%, 0%, 0%);
+  order: 2;
+}
+
+.box-3 {
+  background-color: rgb(60%, 0%, 0%);
+  order: 3;
+}
```

Comment:
Will something change If you delete orders for the first 3 elements? Don't add unnecessary style